Packages/com.chisel.editor/Chisel/Editor/Editor/Keyboard/ChiselKeyboardDefaults.cs

```diff
@@ -15,7 +15,7 @@ public class ChiselKeyboardDefaults
 
         public const string     ShortCutCreateBase      = "Chisel/Create/";
         public const string     ShortCutEditModeBase    = "Chisel/Edit Mode/";
-        
```

Comment:
try to be careful with committing whitespace b/c it'll just complicate the history when trying to find where a certain change has been made